lib/pacemaker/pcmk_sched_instances.c

```diff
@@ -885,69 +885,84 @@ orig_action_name(const pe_action_t *action)
     return task2text(orig_task);
 }
 
+/*!
+ * \brief Update two interleaved actions according to an ordering between them
+ *
+ * Given information about an ordering of two interleaved actions, update the
+ * actions' flags (and runnable_before members if appropriate) as appropriate.
+ * In some cases, the ordering could be disabled.
```

Comment:
Since this is related to the update_ordered_actions() method, and in fact calls it, I went with similar comments.

I believe that refers to pcmk__update_action_for_orderings() and update_action_for_ordering_flags() setting order->type to pe_order_none. However those are actually called for cascaded orderings rather than the ordering here, so the comment is not entirely accurate.